src/aap_eda/tasks/ruleset.py

```diff
@@ -55,24 +55,32 @@ def activate_rulesets(
 
 @job("default")
 def deactivate_rulesets(
+    user_deactivate: bool,
```

Comment:
The name `user_deactivate` is confused, how about changing it to `triggered_by_scheduler`?